system/settings/settings.xml

```diff
@@ -2544,7 +2548,7 @@
           <level>2</level>
           <default>false</default>
           <dependencies>
-            <dependency type="visible">
+            <dependency type="enable">
```

Comment:
:-)

-> added to the famous fernetmenta cite book.

2016-09-11 18:47 GMT+02:00 Rainer Hochecker <notifications@github.com>:

> In system/settings/settings.xml
> <https://github.com/xbmc/xbmc/pull/10451#discussion_r78298353>:
>
> > @@ -2544,7 +2548,7 @@
> >            <level>2</level>
> >            <default>false</default>
> >            <dependencies>
> > -            <dependency type="visible">
> > +            <dependency type="enable">
>
> showing DTSHD on a system that does not support it is the same nonsense as
> showing VAAPI on a NVIdia system. Settings are not a means for educating
> the clueless users.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/10451/files/7d46f87f8312561604d6deea132efa30490a88c8#r78298353>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCfHU_U1fqdEW8E-5hm8k5ER6N_mU7qks5qpDCxgaJpZM4J6Br4>
> .
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B